### PR TITLE
Add more features

### PR DESCRIPTION
**Pull request by HerrNaN**
Add more features

### CODE REVIEW

**Comment by HerrNaN** [comment]
```
! Package xcolor Error: Undefined color `pblue'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.30 func 
          main() {
? 

! Package xcolor Error: Undefined color `pred'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.31     fmt.Println("
                      hello world")
? 

! Package xcolor Error: Undefined color `pred'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.31     fmt.Println("hello 
                            world")
? 

! Package xcolor Error: Undefined color `pred'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.31     fmt.Println("hello 
                            world")
? 

! Package xcolor Error: Undefined color `pred'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.31     fmt.Println("hello world"
                                  )
? 

! Package xcolor Error: Undefined color `pred'.

See the xcolor package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.31     fmt.Println("hello world"
                                  )
? 
[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}] (./test.aux) ){/usr/shar
e/texmf-dist/fonts/enc/dvips/cm-super/cm-super-ts1.enc}</usr/share/texmf-dist/f
onts/type1/public/amsfonts/cm/cmbx10.pfb></usr/share/texmf-dist/fonts/type1/pub
lic/amsfonts/cm/cmbx12.pfb></usr/share/texmf-dist/fonts/type1/public/amsfonts/c
m/cmbxti10.pfb></usr/share/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb>
</usr/share/texmf-dist/fonts/type1/public/amsfonts/cm/cmti10.pfb></usr/share/te
xmf-dist/fonts/type1/public/amsfonts/cm/cmtt10.pfb></usr/share/texmf-dist/fonts
/type1/public/cm-super/sfrm1000.pfb>
Output written on test.pdf (1 page, 66086 bytes).
Transcript written on test.log.

```
(edit: soved with 933a13147672660616a10a4b9a508463d167b2fc)

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+			echo "Installing 'go-bindata'..."; \
+			go get -u github.com/go-bindata/go-bindata/...; \
+		else \
+			echo "go-bindata will not be installed"; exit 1; \
+	fi
 
-# install with go get -u github.com/go-bindata/go-bindata/...
 .PHONY: gen
-gen:
+gen: ${GOPATH}/bin/go-bindata
```
Comment (williamleven) [inline Makefile]: Maybe add a note about setting up your $PATH

**Review by williamleven — APPROVED** [review]

**Comment by williamleven** [comment]
`gobin="$(go env GOBIN)"; [[ -n "$gobin" ]] && echo $gobin || echo "$(go env GOPATH)/bin"` give you the path where go installs packages

File: Makefile (hunk 1 above)
Comment (HerrNaN) [reply Makefile]: Makes sense

**Pull request merged**